docs/userguide/execution.rst

```diff
@@ -168,3 +168,49 @@ at 2 tasks :
 at 5 tasks, we overflow as the capacity of a single block is fully used.
 
 .. image:: parsl_parallelism.gif
+
+
+Multi-Site
+----------
+
+.. note::
+   This feature is available since Parsl 0.4.0
+
+Parsl supports the definition of any number of execution sites in the config,
+as well as specifying which of these sites could execute specific apps.
+
+The common scenarios for this feature are:
+
+* The workflow has an initial simulation stage that runs on the compute heavy
+  nodes of an HPC system followed by an analysis and visualization stage that
+  is suited for the GPU nodes.
+* The workflow follows a repeated fan-out, fan-in model where the long running
+  fan-out tasks are computed on a cluster and the quick fan-in computation is
+  suited for the running on threads on the login node.
+* Have apps that wait and evaluate the results of a computation to determine
```

Comment:
Have apps that wait and evaluate the results of a computation to determine whether, the app should be relaunched.
->
The workflow includes apps that wait and evaluate the results of a computation to determine whether the app should be relaunched.